.gitignore

```diff
@@ -99,3 +99,11 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+.DS_Store
+*.rda
+.vscode/settings.json
+bayesrules/Section_2_1_Fake_news.ipynb
+BCM/*
```

Comment:
Why are all these added to gitignore?

Reply:
To prevent that they appear in the pull request.